main/guides/coreeval/permissions.md

```diff
@@ -0,0 +1,89 @@
+# Coding Permissions
+
+Write a json file with the permissions that the proposal will need. For example, [here](https://github.com/Agoric/agoric-sdk/blob/master/packages/inter-protocol/test/psm/gov-add-psm-permit.json) is the proposal Agoric created for the PSM contract:
+
+
+::: details Show example permissions file
+```
+{
+  "consume": {
+    "agoricNamesAdmin": true,
+    "bankManager": true,
+    "board": true,
+    "chainStorage": true,
+    "zoe": "zoe",
+    "feeMintAccess": "zoe",
+    "economicCommitteeCreatorFacet": "economicCommittee",
+    "econCharterKit": true,
+    "provisionPoolStartResult": true,
+    "psmKit": true,
+    "chainTimerService": "timer"
+  },
+  "produce": {
+    "testFirstAnchorKit": true
+  },
+  "installation": {
+    "consume": {
+      "contractGovernor": true,
+      "psm": true,
+      "mintHolder": true
+    }
+  },
+  "instance": {
+    "consume": {
+      "economicCommittee": true
+    }
+  },
+  "issuer": {
+    "produce": {
+      "DAI": true
+    },
+    "consume": {
+      "DAI": true
+    }
+  },
+  "brand": {
+    "consume": {
+      "DAI": true,
+      "IST": true
+    },
+    "produce": {
+      "DAI": true
+    }
+  }
+}
+```
+:::
+
+## Consume Section
+
+In this section you need to set all the permissions that your contract will need to be able to use (i.e., "consume"). Some of the listed permissions in the example PSM permission file above are of general interest to most contracts, while others are more specific to the PSM contract.
+
+* **agoricNamesAdmin**: Grants write access to the Agoric name service. This permission is somewhat specific to the PSM contract.
+* **bankManager**: Grants access to bank-related functionality within ERTP and Zoe. Most contracts will need access.
+* **board**: Grants write access to the board name service. This permission is somewhat specific to the PSM contract.
+* **chainStorage**: Grants write access to the chain storage node, which is required when running `agd query` commands. Thus, most contracts will need access to this.
+* **zoe**: When this permission is set to "zoe", it grants access to the Zoe framework. All contracts will need access to this.
```

Comment:
It's probably worth mentioning above here that to "set" a permission, its value can be any truthy primitive.  The value itself is not meaningful to the proposal (`true`, `"zoe"`, `"timer"`, `"razzleflatz"` and `249`, are interchangeable and mean the same thing).

As a footnote, Agoric conventionally names its permission values for the identity of the home vat of the ocap corresponding to the permission.  This can be useful for visualizing the produce/consume dependencies between actions.

Then, I'd change all the "permission is set to XXX" just to be "permission is set".

```suggestion
* **zoe**: When this permission is set, it grants access to the Zoe framework. All contracts will need access to this.
```

Reply:
An explanation was added to the top of the Permissions page saying that any string will set a permission. Similarly, all the permission descriptions were simplified per your suggestion above.